src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/CommentsIndentation/newline/defaultContextConfig.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="CommentsIndentation"/>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="context" />
```

Comment:
1. It is a newline strategy why do you use context?
2. Here (https://github.com/checkstyle/patch-filters/pull/168#discussion_r469755922) you commented that every context strategy check needs `supportContextStrategyChecks` property but it looks like it works without this property. Please explain.

Reply:
1. an oversight, corrected.

2. it works because in this case, it is just  a added lines case, so it's ok. in `context` strategy, it will pass all added, patched, deleted and context lines' violation, if users do not use `supportContextStrategyChecks`, then added, patched, deleted lines' violation are still ok, but context lines' violation will be not passed.